File: csharp/easy/fruits_into_baskets_ii.cs
int NumOfUnplacedFruits(int[] fruits, int[] baskets)
{
    int n = baskets.Length;
    int placed = 0;

    foreach (int fruit in fruits)
    {
        for (int i = 0; i < n; i++)
        {
            if (fruit <= baskets[i])
            {
                placed++;
                baskets[i] = 0;
                break;
            }
        }
    }

    return n - placed;
}

File: typescript/easy/fruits-into-baskets-ii.ts
function numOfUnplacedFruitsII(fruits: number[], baskets: number[]): number {
    const n = baskets.length;
    let placed = 0;

    for (const fruit of fruits) {
        for (let i = 0; i < n; i++) {
            if (fruit <= baskets[i]) {
                placed++;
                baskets[i] = 0;
                break;
            }
        }
    }

    return n - placed;
}
